leaf-ui/js/displayAnalysis.js

```diff
@@ -177,144 +177,6 @@ function updateNodeValues(nodeID, satValue) {
     }
 }
 
-/**
```

Comment:
I think you should also be able to delete a lot of the configuration sidebar specific functions as well (for example: switchConfigs, switchSelectedShadingConfig, etc).